### PR TITLE
Get list of unreleased PRs from GitHub

### PR DESCRIPTION
**Pull request by zbynek**
Replace hardcoded list of PRs by API query for
https://github.com/orgs/jenkinsci/projects/3

CC @oleg-nenashev

### CODE REVIEW

File: reports.js [hunk 1]
```diff
@@ -1,14 +1,41 @@
...
-const pulls = require('./pulls.js');
+const axiosGH = require('axios');
 const updatesUrl = 'http://updates.jenkins.io/plugin-documentation-urls.json';
 const installsUrl = 'https://stats.jenkins.io/jenkins-stats/svg/' + lastReportDate() + '-plugins.csv';
 const httpCache = {};
+const gitHubToken = process.env.GITHUB_TOKEN;
+const graphql = 'query {'+
+ ' organization(login:"jenkinsci") {'+
+ '   project(number:3 ) {'+
+ '     columns (first:100) {'+
```
Comment (halkeye) [inline reports.js]: This isn't going to paginate, so no more than like 100 entries I think?

**Comment by halkeye** [comment]
You'll have to contact the infra people to figure out which github account should run this, i can modify the secrets, but other than creating a new one, donno which secret to add

**Comment by timja** [comment]
It probably needs a new GitHub app created and then easiest is Basic auth with its Payton credentials

File: reports.js [hunk 2]
```diff
@@ -1,14 +1,41 @@
+/* eslint-env node */
 const axios = require('./axios');
-const pulls = require('./pulls.js');
+const axiosGH = require('axios');
 const updatesUrl = 'http://updates.jenkins.io/plugin-documentation-urls.json';
 const installsUrl = 'https://stats.jenkins.io/jenkins-stats/svg/' + lastReportDate() + '-plugins.csv';
 const httpCache = {};
+const gitHubToken = process.env.GITHUB_TOKEN;
+const graphql = 'query {'+
```
Comment (timja) [inline reports.js]: Can’t you use a multi line string?

File: reports.js (hunk 1 above)
Comment (zbynek) [reply reports.js]: IIUC this is limiting number of columns, which is curently 4 and won't grow much. Number of fetched cards per column is not limited (for some reason GH enforces `first` for `columns` but not `cards` ...)

Comment (halkeye) [reply reports.js]: Your so right. Thats what my notes have as well (I assume you saw https://gist.github.com/halkeye/4220bc8b0d204af4815d69dd5c9db204)

Comment (zbynek) [reply reports.js]: Your gist is where I got the GraphQL from :)

**Comment by halkeye** [comment]
@olblak @rtyler 

If we don't hear back from either of them by my day tomorrow, I'll just create a new temp bot for now, we can always migrate later.

**Comment by olblak** [comment]
@halkeye, I am pushing a new secret right now, then I'll update this chart, stay tune 

> GITHUB_TOKEN        - token for GitHub API, needs `read:org` and `public_repo` scopes

**Pull request merged**